Assets/Scripts/Data/Pickup Pup/PPDataController.cs

```diff
@@ -54,11 +54,20 @@ public Currency DogFood
 		}
 	}
 
+    public Currency OpenHomeSlots
```

Comment:
I get confused by this var name because of "Open" also being a verb. Looks too much like a function. Can we call it "Unlocked" or "Available" HomeSlots instead?